nx-X11/lib/configure.ac

```diff
@@ -0,0 +1,283 @@
+# Initialize Autoconf
+AC_PREREQ([2.60])
+AC_INIT([libNX_X11], [1.6.5],
+        [https://github.com/ArcticaProject/nx-libs/issues], [libNX_X11])
+AC_CONFIG_AUX_DIR([build-aux])
+AC_CONFIG_SRCDIR([Makefile.am])
+AC_CONFIG_HEADERS([src/config.h include/X11/XlibConf.h])
+AC_CONFIG_MACRO_DIR([m4])
+
+AC_CANONICAL_BUILD
+AC_CANONICAL_HOST
+
+# Set common system defines for POSIX extensions, such as _GNU_SOURCE
+# Must be called before any macros that run the compiler (like AC_PROG_LIBTOOL)
+# to avoid autoconf errors.
+AC_USE_SYSTEM_EXTENSIONS
+
+# Initialize Automake
+AM_INIT_AUTOMAKE([foreign dist-bzip2])
+
+# Initialize libtool
+AC_PROG_LIBTOOL
+
+# Upstream's pkg.m4 (since 0.27) offers this now, but define our own
+# compatible version in case the local version of pkgconfig isn't new enough.
+# https://bugs.freedesktop.org/show_bug.cgi?id=48743
+m4_ifdef([PKG_INSTALLDIR], [PKG_INSTALLDIR],
+         [AC_ARG_WITH([pkgconfigdir],
+                      [AS_HELP_STRING([--with-pkgconfigdir],
+                      [install directory for nxcompshad.pc pkg-config file])],
+                       [],[with_pkgconfigdir='$(libdir)/pkgconfig'])
+          AC_SUBST([pkgconfigdir], [${with_pkgconfigdir}])])
+
+NX_COMPILER_BRAND
+NX_DEFAULT_OPTIONS
+
```

Comment:
> Missing cross-compiling section. Again, we don't do that, but should copy that over and comment it out via dnl.

Nope. If we will do cross-compiling at some time, then we will pull that over. Non-working commented out code I consider confusing here. And libX11's VCS from X.org will probably always be available, at least for some more decades, I presume... ;-)